### PR TITLE
license: clarify descriptions

### PR DESCRIPTION
**Pull request by waldyrious**
Following discussion in #711

### CODE REVIEW

File: pages/common/license.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 > Create license files for open-source projects.
 
-- Create a license:
+- Print a license to stout, using the defaults (auto-detected author name, and current year):
```
Comment (agnivade) [inline pages/common/license.md]: stout => stdout :smile:  I am good with stout too by the way :stuck_out_tongue_closed_eyes:

Comment (waldyrious) [inline pages/common/license.md]: Hahah nice catch! I must have been thinking of the dinner I'm planning for tonight :P

Comment (ostera) [inline pages/common/license.md]: 🍻

**Comment by ostera** [comment]
LGTM!

**Pull request merged**